gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -1899,4 +1898,30 @@ private String getXAttrName(String key) {
   private byte[] getXAttrValue(byte[] value) {
     return value == null ? XATTR_NULL_VALUE : value;
   }
+
+  /** Override superclass so as to add statistic collection. */
+  @Override
+  public boolean exists(Path f) throws IOException {
+    return super.exists(f);
+  }
+
+  /** Override superclass so as to add statistic collection. */
+  @Override
+  public RemoteIterator<LocatedFileStatus> listLocatedStatus(final Path f)
+      throws FileNotFoundException, IOException {
+    return super.listLocatedStatus(f);
+  }
+
+  /**
+   * Get the instance's instrumentation.
+   *
+   * @return
+   */
+  public GhfsInstrumentation getInstrumentation() {
+    return null;
+  };
+
+  Boolean getIsClosed() {
+    return isClosed();
+  }
```

Comment:
Do we need this in Base class ? Can we move them to Instrumentation class ?

Reply:
Updated